README.adoc

```diff
@@ -285,6 +285,102 @@ spec:
 ...
 ----
 
+== Configure Health
```

Comment:
I am not sure this feature belongs to README.adoc but kiali.io
If we document a feature, we need to document ALL features here.

So, I prefer a better documentation in kiali.io and leave README.adoc more clean (or point to the proper kiali.io place).

Reply:
ok